### PR TITLE
Material

### PR DESCRIPTION
**Pull request by TheMcMurder**
I'm bringing in material-ui because I'm lazy and I'm tired of fighting the old version of the styleguide. Lots of changes coming.

From:
Screenshot from 2019-05-29 15-04-46
To: 
Screenshot from 2019-05-29 15-04-57

And

From: 
Screenshot from 2019-05-29 15-11-10
To:
Screenshot from 2019-05-29 15-11-15

### CODE REVIEW

File: tmeister/static/index.html [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
-		</div>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+    <link rel="shortcut icon" href="https://cdn.canopytax.com/images/canopy-favicon-new-green.jpg">
+  </head>
+  <body id="cps-app">
+    <div id="main-content">
+    </div>
     <script type="text/javascript" src="build/toggle-meister.js"></script></body>
-	</body>
```
Comment (TheMcMurder) [inline tmeister/static/index.html:17]: Tabs -> spaces

File: tmeister/static/package.json [hunk 2]
```diff
@@ -33,8 +33,7 @@
     "webpack-dev-server": "^3.4.1"
   },
   "dependencies": {
-    "cpr-multiselect": "^3.3.14",
-    "cpr-tooltip": "^1.2.1",
+    "@material-ui/core": "^4.0.1",
```
Comment (TheMcMurder) [inline tmeister/static/package.json:36]: material provides both of these

File: tmeister/static/src/root.routes.js [hunk 3]
```diff
@@ -3,18 +3,28 @@ import { Router, Route, Link, Redirect, hashHistory } from 'react-router';
 import TogglesContainer from './toggles/toggles.container.js';
 import RootContainer from './menu/root.container.js';
 import AuditTrail from './audit-trail/audit-trail.container.js';
+import { createMuiTheme } from '@material-ui/core/styles';
+import { ThemeProvider } from '@material-ui/styles'
+
+const palette = {
+  primary: { main: '#00BF4B' },
+  secondary: { main: '#3399ff' }
+};
```
Comment (TheMcMurder) [inline tmeister/static/src/root.routes.js:15]: Using our colors for the theme

**Review by nhumrich — APPROVED** [review]
LGTM

File: tmeister/static/src/root.routes.js (hunk 3 above)
Comment (blittle) [reply tmeister/static/src/root.routes.js:15]: You should probably add a contrast text color:
```
primary: {
  main: '#00BF4B',
  contrastText: '#ffffff',
},
```

File: tmeister/static/src/menu/main-menu.component.js [hunk 4]
```diff
@@ -1,24 +1,62 @@
...
-			<CprTooltip text="Audit trail">
-				<a className={`${styles.menuItem} ${activeIcon("audit-trail")}`} href="/#/audit-trail">
-					<img src='/resources/audit-icon.png' className={`${styles.icon}`} />
-				</a>
-			</CprTooltip>
-		</div>
-	);
+  const classes = useStyles()
+  return (
+    <Paper className={classes.menu} classes={classes}>
```
Comment (blittle) [inline tmeister/static/src/menu/main-menu.component.js:39]: NIT: passing `classes` seems a little weird 🤷‍♀️

File: tmeister/static/src/toggles/create/create-env/create-env.component.js [hunk 5]
```diff
@@ -1,17 +1,18 @@
...
+import Button from '../../../common/simple-button/simple-button.component.js'
 
 export default function CreateEnv (props) {
   const { refetchToggles } = props
   const [showCreateModal, setShowCreateModal] = useState(false)
   return (
     <div>
-      <button
+      <Button
         className="cps-btn +secondary"
```
Comment (blittle) [inline tmeister/static/src/toggles/create/create-env/create-env.component.js]: `className` still necessary?

File: tmeister/static/src/menu/main-menu.component.js (hunk 4 above)
Comment (TheMcMurder) [reply tmeister/static/src/menu/main-menu.component.js:39]: I was surprised by it as well: https://material-ui.com/api/paper/#props

**Pull request merged**